7_react/react1/week3/todo-app/src/component/TodoContainer.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
good job with using bootstrap :) but it's better to import it directly in `index.js` or `App.jsx `then you can use it everywhere :)  https://react-bootstrap.netlify.app/docs/getting-started/introduction/#css